tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -2,13 +2,12 @@
 
 from constants import TRUST_STORE_BUNDLE
```

Comment:
Oh I'm also noticing that permissions changed on this file-- was that intentional?

Reply:
no, also reverted.